src/widget/core.rs

```diff
@@ -224,6 +224,14 @@ impl Pod {
                     false
                 }
             }
+            RawEvent::MouseLeft() => {
+                Pod::set_hot_state(&mut self.widget, &mut self.state, cx.cx_state, rect, None);
+                if had_active || self.state.flags.contains(PodFlags::IS_HOT) {
+                    false
+                } else {
+                    true
+                }
```

Comment:
[Here](https://github.com/linebender/druid/blob/ac3815114c65d46fd388431d3013a9412501916b/druid/src/core.rs#L687)  is druids implementation of the `MouseLeave` event. 
We should probably stay close to that version. Every active or previously hot widget should receive the event.

Reply:
🤦‍♂️ yes that makes a lot more sense.